src/test/java/com/wikia/webdriver/testcases/mercurytests/old/curatedcontenttests/EditorTests.java

```diff
@@ -12,212 +14,118 @@
   private static final String SEARCH_IMAGE_QUERY = "U";
 
-  private CuratedMainPagePageObject curatedMainPage;
-  private CuratedContentPageObject curatedContent;
-  private SectionFormPageObject section;
-  private SectionItemListPageObject sectionItem;
-  private CategoryFormPageObject category;
-  private EditorHomePageObject editor;
-  private ItemFormPageObject itemForm;
-  private UploadImageModalComponentObject uploadImage;
-  private SearchForImagePageObject search;
-  private CroppingToolPageObject croppingTool;
-  private Navigate navigate;
-
-  private void init() {
-    this.curatedMainPage = new CuratedMainPagePageObject(driver);
-    this.curatedContent = new CuratedContentPageObject(driver);
-    this.section = new SectionFormPageObject(driver);
-    this.sectionItem = new SectionItemListPageObject(driver);
-    this.category = new CategoryFormPageObject(driver);
-    this.editor = new EditorHomePageObject(driver);
-    this.itemForm = new ItemFormPageObject(driver);
-    this.uploadImage = new UploadImageModalComponentObject(driver);
-    this.search = new SearchForImagePageObject(driver);
-    this.croppingTool = new CroppingToolPageObject(driver);
-    this.navigate = new Navigate();
-  }
-
   @BeforeMethod(alwaysRun = true)
   public void beforeMethod() {
     new CuratedContent().clear();
   }
 
-  @AfterClass(alwaysRun = true)
-  public void afterClass() {
-    new CuratedContent().clear();
-  }
-
   @Test(groups = "MercuryCuratedEditorTest_001")
   public void MercuryCuratedEditorTest_001_addAndSaveItemToFeaturedContent() {
-    init();
-
-    navigate.toPage(MercurySubpages.ECC_MAIN_PAGE);
-    Assertion.assertTrue(new SkinHelper(driver).isSkin(Skin.MOBILE_WIKI));
-
-    Boolean result = curatedMainPage.isFeaturedContentVisible();
-
-    Assertion.assertFalse(result, MercuryMessages.VISIBLE_MSG);
-
-    navigate.toPage(MercuryPaths.ROOT_MAIN_EDIT);
-    Assertion.assertTrue(new SkinHelper(driver).isSkin(Skin.MERCURY));
-
-    editor.clickAddFeaturedContent();
-    itemForm.typeDisplayName(ITEM_DISPLAY_NAME);
-    itemForm.typePageName(ITEM_PAGE_NAME);
-    itemForm.clickOnImage();
-    uploadImage.clickSearchForImageButton();
-    search.type(SEARCH_IMAGE_QUERY);
-    search.clickOnImage(0);
-    croppingTool.clickDoneButton();
-
-    itemForm.waitForDeleteButtonToBeVisible();
-    itemForm.clickDone();
-
-    editor.waitForAddCategoryButtonToBeVisible();
-    editor.publish();
-
-    Assertion.assertTrue(new SkinHelper(driver).isSkin(Skin.MOBILE_WIKI));
-    result = curatedMainPage.isFeaturedContentVisible();
-
-    PageObjectLogging.log(
-        "Featured Content",
-        MercuryMessages.VISIBLE_MSG,
-        MercuryMessages.INVISIBLE_MSG,
-        result
-    );
+    Boolean result = new ArticlePage().open(MercurySubpages.ECC_MAIN_PAGE).getCurratedMainPage()
+        .isFeaturedContentVisible();
+
+    PageObjectLogging.logInfo(String.format("Curated content is visible: %s", result));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Curated content is visible: %s" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)